packages/nimble-components/src/select/specs/filterable-select-hld.md

```diff
@@ -88,25 +128,19 @@ The accessibility tree will report that the search `input` element should have i
 
 ### Future considerations
 
-#### Grouping/Metadata
+It's possible that we may want to improve/alter the discoverability of the fact that more options are available to be loaded when using the `manual` filtering option. This _could_ follow the design of what we see with the AzDO _Windows_-based user selector experiece where a display of the number of current results is shown at the bottom, and when a user begins typing in the filter input, a 'Show more results' button is shown at the bottom of the dropdown.
```

Comment:
> the AzDO _Windows_-based user selector experiece

- typo in "experience"
- what do you mean by Windows-based? Is this not available in the web UI?
- all this background seems of questionable value. I'd vote to replace it with something more vague ("For example, the dropdown could include a "Show more results button"), but if you want to keep it then maybe a screenshot would be more helpful to explain it.

Reply:
I've removed the description for a possible solution, and instead just outlined the problem a bit more clearly.